apis/core/package.json

```diff
@@ -46,7 +46,7 @@
     "express-jwt-permissions": "^1.3.3",
     "express-rdf-request": "0.1.0",
     "http-errors": "^1.8.0",
-    "hydra-box": "zazuko/hydra-box#subpath-api-link",
```

Comment:
I couldn't find this branch in the hydra-box repo. Has it been merged since?

Reply:
https://github.com/zazuko/hydra-box/commit/00685acab3475d6329ee46ed914ef4f32204c68b

Reply:
Good, should be fine then